hips/hip-0020.md

```diff
@@ -0,0 +1,133 @@
+---
+hip: "0020"
+title: "H4HIP: Charts v3 Enablement"
+authors: [ "Matt Farina <matt.farina@suse.com>" ]
+created: "2025-01-09"
+type: "feature"
+status: "accepted"
+---
+
+## Abstract
+
+This HIP proposes the creation of charts v3, updating Helm to handle charts v2 and v3, and a
+timeline for the general availability of charts v3 that can happen after the release of Helm v4.0.0.
```

Comment:
Should we explicitly call out this HIP introduces `apiVersion: v3`?

Reply:
Same as https://github.com/helm/community/pull/381#discussion_r1960499966, I think we can detail that in the docs, blog post, etc.